src/MongoDB/Logging/Logger.stub.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @generate-class-entries static
+ * @generate-function-entries
+ */
+
+namespace MongoDB\Driver\Logging;
+
+interface Logger
+{
+    /**
+     * @var int
+     * @cvalue MONGOC_LOG_LEVEL_ERROR
+     */
+    public const LEVEL_ERROR = UNKNOWN;
+
+    /**
+     * @var int
+     * @cvalue MONGOC_LOG_LEVEL_CRITICAL
+     */
+    public const LEVEL_CRITICAL = UNKNOWN;
+
+    /**
+     * @var int
+     * @cvalue MONGOC_LOG_LEVEL_WARNING
+     */
+    public const LEVEL_WARNING = UNKNOWN;
+
+    /**
+     * @var int
+     * @cvalue MONGOC_LOG_LEVEL_MESSAGE
+     */
+    public const LEVEL_MESSAGE = UNKNOWN;
```

Comment:
Are we OK with adding these constants on the Logger interface, or would it make sense to push these to their own interface similar to [Psr\Log\LogLevel](https://www.php-fig.org/psr/psr-3/#5-psrlogloglevel)?

I could go either way, but lean towards the side of consolidating our API/docs footprint (i.e. less classes).

Reply:
I think these can stay in as constants, to be deprecated when we bump the PHP version enough to create a proper `enum` for the log levels.